sqlx-sqlite/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "sqlx-sqlite-index-example"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+anyhow = "~1"
+arrow = "51.0.0"
+async-trait = "0.1.80"
+datafusion = { features = ["backtrace"], git = "https://github.com/adriangb/datafusion.git" }
+datafusion-expr = { git = "https://github.com/adriangb/datafusion.git" }
+datafusion-optimizer = { git = "https://github.com/adriangb/datafusion.git" }
+datafusion-common = { git = "https://github.com/adriangb/datafusion.git" }
+datafusion-physical-expr = { git = "https://github.com/adriangb/datafusion.git" }
+datafusion-sql = { git = "https://github.com/adriangb/datafusion.git" }
```

Comment:
This is just `main` with https://github.com/apache/datafusion/pull/10813 merged

Reply:
I just merged https://github.com/apache/datafusion/pull/10813 now too (it will be in datafusion 40.0.0, not 39.0.0)